### PR TITLE
Dump module items

### PR DESCRIPTION
**Pull request by CohenArthur**

### CODE REVIEW

File: gcc/rust/ast/rust-ast-dump.cc [hunk 1]
```diff
@@ -554,7 +579,23 @@ Dump::visit (Method &method)
 
 void
 Dump::visit (Module &module)
-{}
+{
+  indentation.increment ();
```
Comment (jdupak) [inline gcc/rust/ast/rust-ast-dump.cc:583]: I feel this should be after the curly brace (line 588), to indent the body only.

Comment (jdupak) [reply gcc/rust/ast/rust-ast-dump.cc:583]: Also a space after mod is missing.

Comment (CohenArthur) [reply gcc/rust/ast/rust-ast-dump.cc]: Fixed in [6bb6550](https://github.com/Rust-GCC/gccrs/pull/1548/commits/6bb6550c44d6644bb22e4a66be467802faeb23d8)

**Comment by jdupak** [comment]
I have just noticed that this implementation is incomplete. It ignores the module kind case - external - which should not have curly braces. I will fix that in a followup PR.

**Comment by jdupak** [comment]
(https://doc.rust-lang.org/reference/items/modules.html#the-path-attribute)

**Comment by CohenArthur** [comment]
> I have just noticed that this implementation is incomplete. It ignores the module kind case - external - which should not have curly braces. I will fix that in a followup PR.

Perfect, thanks a lot. I had completely forgotten!

**Pull request merged**